docs/src/getting_started/Android/android.md

```diff
@@ -47,7 +51,7 @@ elided):
 │  ├── build.rs
 │  ├── Cargo.toml
 │  ├── src
-│  │  ├── counter.rs
+│  │  ├── app.rs
```

Comment:
I think this is just drift -- the book has this as `app.rs` and `simple_counter` earlier -- https://redbadger.github.io/crux/getting_started/core.html#the-app and https://redbadger.github.io/crux/getting_started/core.html#the-interface-definitions

Reply:
yeah it is! Good catch, Thanks